### PR TITLE
Update Chinese

### PR DESCRIPTION
**Pull request by Jesse205**
- Use of Chinese symbols
- Correct use of punctuation (major fixes for incorrect use of `, ` and missing `. `)
- Make semantics of some strings clearer

### CODE REVIEW

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -63,8 +63,8 @@
     <string name="content">内容</string>
     <string name="logs_format">日志格式</string>
     <string name="no_permission_detected">未获取权限</string>
-    <string name="logs_restart_required">所有日志在重启后才能获得</string>
-    <string name="too_big_log">日志太大,尝试压缩后发送</string>
+    <string name="logs_restart_required">所有日志将在重启后才可获得</string>
```
Comment (huajijam) [inline app/src/main/res/values-zh-rCN/strings.xml:66]:
```suggestion
    <string name="logs_restart_required">所有日志只有在重启后才可用</string>
```
ref to RUS:"Все логи будут доступны только после перезапуска"

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 2]
```diff
@@ -78,37 +78,37 @@
...
     <string name="start_service">启动服务</string>
     <string name="stop_service">停止服务</string>
-    <string name="service_required_desc">需要日志服务以在后台获取崩溃情况,如果没有服务, Android 会杀死 LogFox</string>
+    <string name="service_required_desc">需要日志服务以在后台获取崩溃情况，如果没有服务，Android 会杀死 LogFox</string>
     <string name="recording">录制中</string>
     <string name="title">标题</string>
     <string name="recording_status">录制状态</string>
     <string name="warning">警告</string>
-    <string name="recording_with_no_service_warning">你正试图在没有运行日志服务的情况下进行录制,由于 Andorid 的后台应用限制,这可能导致无法保存录制,最好在录制时启动日志服务</string>
+    <string name="recording_with_no_service_warning">你正试图在没有运行日志服务的情况下进行录制。由于 Andorid 的后台应用限制，这可能导致无法保存录制。在录制时启动日志服务更好。</string>
```
Comment (huajijam) [inline app/src/main/res/values-zh-rCN/strings.xml:86]:
```suggestion
    <string name="recording_with_no_service_warning">你正试图在没有运行日志服务的情况下进行录制，由于 Andorid 的后台应用限制，这可能导致无法保存录制，最好在录制时运行日志服务。</string>
```
Semantics are changed

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 3]
```diff
@@ -15,9 +15,9 @@
     <string name="app_crashed">%s 崩溃了</string>
     <string name="start_on_boot">开机时自启</string>
     <string name="filter">过滤器</string>
-    <string name="about_app">关于本App</string>
+    <string name="about_app">关于本 App</string>
     <string name="setup">设置</string>
-    <string name="how_to_use_adb">自从 Android 13 Google 限制了在 ADB 中获取日志,你只能在几分钟内看到日志,之后你需要重启日志服务\n\n在 ADB 中运行该命令:\n\n%s</string>
+    <string name="how_to_use_adb">自从 Android 13 Google 限制了在 ADB 中获取日志。你只能在几分钟内看到日志，之后你需要重启日志服务。\n\n在 ADB 中运行该命令：\n\n%s</string>
```
Comment (huajijam) [inline app/src/main/res/values-zh-rCN/strings.xml:20]:
```suggestion
    <string name="how_to_use_adb">自 Android 13，Google 限制了通过 ADB 收集日志。你只能看到几分钟的日志，然后需要重启日志服务。\n\n在 ADB 中运行此命令：\n\n%s</string>
```

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 4]
```diff
@@ -78,37 +78,37 @@
...
+    <string name="new_terminal_selected_question">是否使用该后端重启日志记录？</string>
     <string name="yes">是</string>
     <string name="no">否</string>
     <string name="restart_logging">重启日志记录</string>
     <string name="shizuku_error">使用 Shizuku 时遇到错误</string>
-    <string name="shizuku_reminder">别忘了启动 Shizuku!</string>
+    <string name="shizuku_reminder">别忘了启动 Shizuku！</string>
     <string name="fallback_to_default_terminal">回退到默认后端</string>
-    <string name="fallback_to_default_terminal_desc">仅当所选后端不可用时,否则 LogFox 将一直尝试使用所选的</string>
+    <string name="fallback_to_default_terminal_desc">仅当所选后端不可用时才会回退，否则 LogFox 将一直尝试使用所选的后端</string>
```
Comment (huajijam) [inline app/src/main/res/values-zh-rCN/strings.xml:106]:
```suggestion
    <string name="fallback_to_default_terminal_desc">仅当所选后端不可用时回退，否则 LogFox 将一直尝试使用所选后端</string>
```

**Review by huajijam — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-zh-rCN/strings.xml (hunk 2 above)
Comment (Jesse205) [reply app/src/main/res/values-zh-rCN/strings.xml:86]: I don't think the semantics have changed. These are two completely different sentences, so you need to use `。` to separated

Comment (Jesse205) [reply app/src/main/res/values-zh-rCN/strings.xml:86]: I don't think the semantics have changed. These are two completely different sentences, so there need to use `。` to separated

Comment (Jesse205) [reply app/src/main/res/values-zh-rCN/strings.xml:86]: I don't think the semantics have changed. These are two completely different sentences, so you need to use `。` to separated

Comment (Jesse205) [reply app/src/main/res/values-zh-rCN/strings.xml:86]: I don't think the semantics have changed. These are two completely different sentences, so there need to use `。` to separated

Comment (Jesse205) [reply app/src/main/res/values-zh-rCN/strings.xml:86]: 我看不出有什么语义发生的变化。相反我的修改可以更委婉地表达出原意，你的这个修改有非常奇怪的感觉。

I can't see any semantic changes happening. On the contrary, my modifications can express the original intention more tactfully. Your modification has a very strange feeling.

Comment (huajijam) [reply app/src/main/res/values-zh-rCN/strings.xml:86]:
```suggestion
    <string name="recording_with_no_service_warning">你正试图在没有运行日志服务的情况下进行录制。由于 Andorid 的后台应用限制，这可能导致无法保存录制，最好在录制时运行日志服务。</string>
```

Comment (huajijam) [reply app/src/main/res/values-zh-rCN/strings.xml:86]: 语气强度的区别
请检查RUS用词

Comment (huajijam) [reply app/src/main/res/values-zh-rCN/strings.xml:86]: 这是一个warning 不是一个hint

**Pull request merged**